pkg/controller/master/image/controller.go

```diff
@@ -97,7 +97,19 @@ func (h *Handler) OnImageChanged(key string, image *apisv1alpha1.Image) (*apisv1
 func (h *Handler) OnImageRemove(key string, image *apisv1alpha1.Image) (*apisv1alpha1.Image, error) {
 	logrus.Debugf("removing %s object in minio", image.Name)
 	cleanUpImport(image.Name, image.Status.AppliedURL)
-	return image, util.MinioClient.RemoveObject(util.BucketName, image.Name)
+	if err := removeImageFromStorage(image); err != nil {
+		// Log the error and do not block resource deletion.
+		logrus.Errorf("failed to remove image %q from storage: %v", image.Spec.DisplayName, err)
+	}
+	return image, nil
```

Comment:
Why cannot the user remove the image from storage? is it possible to return the error message to the frontend but not just log in the backend?